bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/handler/RemoteopenhabBridgeHandler.java

```diff
@@ -484,108 +484,108 @@ private void updateChannelState(String itemName, @Nullable String stateType, Str
-                        switch (state) {
-                            case "PLAY":
-                                channelState = PlayPauseType.PLAY;
-                                break;
-                            case "PAUSE":
-                                channelState = PlayPauseType.PAUSE;
-                                break;
-                            default:
-                                logger.debug("Unexpected value {} for item {}", state, itemName);
-                                break;
+                        break;
+                    case CoreItemFactory.LOCATION:
+                        if (checkStateType(itemName, stateType, "Point")) {
+                            channelState = new PointType(state);
                         }
-                    }
-                    break;
-                case CoreItemFactory.ROLLERSHUTTER:
-                    if (checkStateType(itemName, stateType, "Percent")) {
-                        channelState = new PercentType(state);
-                    }
-                    break;
-                default:
-                    logger.debug("Item type {} is not yet supported", acceptedItemType);
-                    break;
+                        break;
+                    case CoreItemFactory.IMAGE:
+                        if (checkStateType(itemName, stateType, "Raw")) {
+                            channelState = RawType.valueOf(state);
+                        }
+                        break;
+                    case CoreItemFactory.PLAYER:
+                        if (checkStateType(itemName, stateType, "PlayPause")) {
+                            switch (state) {
+                                case "PLAY":
+                                    channelState = PlayPauseType.PLAY;
+                                    break;
+                                case "PAUSE":
+                                    channelState = PlayPauseType.PAUSE;
+                                    break;
+                                default:
+                                    logger.debug("Unexpected value {} for item {}", state, itemName);
+                                    break;
+                            }
+                        }
+                        break;
+                    case CoreItemFactory.ROLLERSHUTTER:
+                        if (checkStateType(itemName, stateType, "Percent")) {
+                            channelState = new PercentType(state);
+                        }
+                        break;
+                    default:
+                        logger.debug("Item type {} is not yet supported", acceptedItemType);
+                        break;
+                }
             }
+        } catch (IllegalArgumentException | DateTimeException e) {
+            logger.warn("Failed to parse state \"{}\" for item {}: {}", state, itemName, e.getMessage());
+            channelState = null;
```

Comment:
Shouldn't it set the state to `UNDEF`? to indicate something is wrong. By setting it to null it seems to be used previously as a completely unsupported and thus nothing was updated (maybe in cases that could theoretically not happen)

Reply:
Looks like a good idea.

Reply:
Done